mobileapps/views.py

```diff
@@ -388,6 +387,57 @@ def delete(self, request, mobile_app_id):
             raise Http404
 
 
+class MobileAppsNotifications(MobileAPIView):
+    """
+    **Use Cases**
+
+        send a push notification to all the users of all active apps
+        using app's push notifications provider.
+
+    **Example Requests**
+
+        POST /api/server/mobileapps/notification
+
+        The body of the POST request must include the following parameters.
+
+        * message: notification message
+
+    **Response Values**
+
+        If the request is successful, the request returns an HTTP 202 "Accepted" response.
+
+        The HTTP 202 response has the following value.
+
+        * message: Accepted
+    """
+
+    def post(self, request):
+
+        message = request.data.get('message', None)
+        if not message:
+            return Response({'message': _('message is missing')}, status.HTTP_400_BAD_REQUEST)
+        payload = {
+            'title': message,
+            'send_to_all': True
+        }
+
+        try:
+            mobile_apps = MobileApp.objects.filter(is_active=True)
```

Comment:
nit: what if we add `notification_provider__isnull=False` filter here to avoid the `if` statement below?.

Reply:
makes sense. let me implement this one